librz/bin/bin.c

```diff
@@ -673,6 +675,33 @@ RZ_API RZ_BORROW RzBinMap *rz_bin_object_get_map_at(RZ_NONNULL RzBinObject *o, u
 	return NULL;
 }
 
+/**
+ * \brief Find the binary symbol at offset \p off.
+ *
+ * This function returns the binary symbol that contains offset \p off.
+ * It iterates over the list of symbols and directly compares the offset with the virtual or physical address.
+ *
+ * \param o Reference to the \p RzBinObject instance
+ * \param off Address to search
+ * \param va When false, the offset \p off is considered a physical address; otherwise, a virtual address
+ * \return Pointer to a \p RzBinSymbol containing the address, or NULL if no symbol is found at the address
+ */
+RZ_API RZ_BORROW RzBinSymbol *rz_bin_object_get_symbol_at(RZ_NONNULLL RzBinObject *o, ut64 off, bool va) {
```

Comment:
this is still missing in the header. also `RZ_NONNULLL`. please compile the solution before pushing changes.